http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -514,8 +517,14 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         // general configs
         raw"fullnameOverride=$appChartPrefix-${release.asString}",
         raw"instrumentationEnabled=${config.coaAppConfig.instrumentationEnabled}"
-      ).mkString(",")
-    )
+      )
+
+    val updatedLs = sourceWorkspaceId match { // TODO remove after WDS chart migration
+      case Some(value) => valuesList ::: List(raw"provenance.sourceWorkspaceId=${value.value}")
+      case None        => valuesList
+    }
```

Comment:
This is perfectly fine as-is, no need to change anything. For the sake of discussing Scala goodness, some thoughts:

You could rewrite this using `.map`, which is a bit more functional-programming-esque than an explicit `match` statement here:
```scala
val updatedLs = valuesList ++ sourceWorkspaceId.map(v => raw"provenance.sourceWorkspaceId=${v.value}").toList
```
This also relies on the behavior of `Option.toList` to create an empty list for None and a single-value list for Some.

Also, beware the difference between `:::` and `++` when concatenating lists. The former prepends the right-hand value to the list, while the latter appends. In this particular case I don't think it matters at all, but in other cases it certainly could, and might lead to confusion. At lists of this size the performance impact should be nonexistent.

Reply:
+1 David's suggestion is a bit more idiomatic, but also ok as is.

Does the helm chart handle no value for `provenance.sourceWorkspaceId` if the app is not a clone? E.g. it doesn't need some default value?

Reply:
In my testing, it was injecting the placeholder value - i.e. "RUNTIME_PARAMETER", so I used a blank string "".  I'm happy to hear a better solution.